srcpkgs/veikk-tablet/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'veikk-tablet'
+pkgname=veikk-tablet
+version=3.5.8
+_pkgname=vktablet
+revision=3
+_pkgv=${version}-${revision}
+archs="x86_64"
+short_desc="Veikk tablet driver retrieved from official Veikk website"
+homepage="https://www.veikk.com"
+license="custom:Properietary"
+maintainer="Hamdan Dirgantoro <hamdandirgantara@gmail.com"
+restricted=yes
+repository=nonfree
+distfiles="https://veikk.com/image/catalog/Software/${_pkgname}-${_pkgv}.${archs}.zip"
```

Comment:
neither the `archs` or `_pkgname` variables are required here, they are not actually variable, so just "hardcode" the values.